packages/design-picker/src/components/theme-preview-container/style.scss

```diff
@@ -0,0 +1,104 @@
+@import '@automattic/onboarding/styles/mixins';
+
+.theme-preview-container {
+	display: flex;
+	flex-direction: row;
+}
+
+.theme-preview-container__sidebar {
+	width: 343px;
```

Comment:
I think it needs to have horizontal padding `32px` or we could just set the width to be `280px`